bi_sql_editor/models/bi_sql_view.py

```diff
@@ -562,7 +547,7 @@ def _create_model_and_fields(self):
             sql_view.rule_id = self.env["ir.rule"].create(self._prepare_rule()).id
             # Drop table, created by the ORM
             if sql.table_exists(self._cr, sql_view.view_name):
-                req = "DROP TABLE %s" % sql_view.view_name
+                req = f"DROP TABLE {sql_view.view_name}"
```

Comment:
Better to use `sql.SQL` and `sql.Identifier` from `psycopg2` instead to avoid SQL injection, see https://www.psycopg.org/docs/sql.html